sunpy/map/tests/test_mapbase.py

```diff
@@ -1101,3 +1101,8 @@ def test_contour(simple_map):
     assert contour.obstime == simple_map.date
     assert u.allclose(contour.Tx, [0, -1, 0, 1, 0] * u.arcsec, atol=1e-10 * u.arcsec)
     assert u.allclose(contour.Ty, [0.5, 0, -0.5, 0, 0.5] * u.arcsec, atol=1e-10 * u.arcsec)
+
+
+def print_map(generic_map):
+    out = generic_map.__repr__()
+    assert isinstance(out, str)
```

Comment:
This test needs expanding to test the print.

Reply:
ok

Reply:
ok